src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -6,7 +6,7 @@
 using System.Runtime.InteropServices;
 using System.Runtime.Intrinsics.X86;
 
-using Internal.Runtime.CompilerServices;
+using Internal.Runtime.CompilerServices; // Unsafe.AddByteOffset
```

Comment:
Nit: We do not add commands on usings like this. They are guaranteed to get out of date very fast.